Resources/Private/Partials/Application/Field/Checkbox.html

```diff
@@ -1,6 +1,6 @@
 <div class="form-group">
 		<div class="col-sm-3">
-			<label><f:translate key="{translationKey}" /></label>
+			<label><f:translate key="{translationKey}" /><f:if condition="{settings.validation.{fieldName}}">*</f:if></label>
```

Comment:
Maybe move the label and asterisk to a partial to avoid repeating this over and over again? Also at least a space should be put between label and asterisk. And maybe wrap the asterisk with a `<span class="required">` (or something Bootstrap-ish) or similar to make small styling customizations easier.

Reply:
Done.